package.json

```diff
@@ -6,6 +6,7 @@
     "dev": "bun run --watch src/index.ts"
   },
   "dependencies": {
+    "@elysiajs/cors": "^0.7.1",
     "elysia": "latest"
```

Comment:
Personally, I would use specific versions.  Using `latest` it will upgrade without checking what changed _could_ cause unexpected behaviors in your application.